packages/advjs/src/client/components/internals/AdvGame.vue

```diff
@@ -29,6 +25,20 @@ if (!isDev)
 
 const app = useAppStore()
 
+const space = (e: KeyboardEvent) => {
+  if (e.code === 'Space') {
+    if (!app.showHistory && !app.showSaveMenu && !app.showLoadMenu)
+      adv.next()
+  }
+}
+onBeforeMount(() => {
```

Comment:
I think you should use [useMagicKeys](https://vueuse.org/core/useMagicKeys/) to simplify this code.